### PR TITLE
Fix bug with passification of neuron

### PR DESCRIPTION
**Pull request by WillemWybo**
fix bug where channel_storage was not emptied when a neuron is passified

### CODE REVIEW

File: tests/test_compartmentfitter.py [hunk 1]
```diff
@@ -614,10 +616,10 @@ def testParallel(self, w_benchmark=False):
 
 if __name__ == '__main__':
     tcf = TestCompartmentFitter()
-    tcf.testTreeStructure()
+    # tcf.testTreeStructure()
```
Comment (jakobj) [inline tests/test_compartmentfitter.py:619]: why do you comment out tests? ;)

File: tests/test_compartmentfitter.py [hunk 2]
```diff
@@ -614,10 +616,10 @@ def testParallel(self, w_benchmark=False):
...
     # tcf.testCreateTreeGF()
     # tcf.testChannelFitMats()
-    # tcf.testPassiveFit()
+    tcf.testPassiveFit()
```
Comment (jakobj) [inline tests/test_compartmentfitter.py:622]: is this testing the proposed changes?

**Review by jakobj — COMMENTED** [review]
too little info to give a more detailed review, just added a two questions ;)

File: tests/test_compartmentfitter.py (hunk 1 above)
Comment (WillemWybo) [reply tests/test_compartmentfitter.py:619]: This is just for testing specific tests, with pytest, they all run anyway.

File: tests/test_compartmentfitter.py (hunk 2 above)
Comment (WillemWybo) [reply tests/test_compartmentfitter.py:622]: Had to debug the test.

**Pull request merged**